tests/scalers_go/gcp_stackdriver/gcp_stackdriver_test.go

```diff
@@ -172,78 +192,113 @@ func TestScaler(t *testing.T) {
 	assert.True(t, sdkReady, "gcp-sdk deployment should be ready after a minute")
 
 	if sdkReady {
-		if createBucket(t) == nil {
+		if createPubsub(t) == nil {
```

Comment:
is `createPubsub` the correct name in stackdriver? (TBH, I don't know what stackdriver is, I'm a real noob with google cloud)

Reply:
Stackdriver is the metrics system. I'm generating here pubsub traffic and use a stackdriver query to monitor it

Reply:
Thanks for the clarification, nothing to add in that case 🙇